components/library/zeromq/zeromq5/zeromq.p5m

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
That is an interesting point... I don't really know.

On one hand, in my testing of building and installing all these packages - the copy of `zeromq.p5m` that has the highest IPS_COMPONENT_VERSION (newest zeromq version number) wins and is the one that gets published and installed, and so "pkg install zeromq" pulls the package with highest number. So from limited practice, it does not hurt to have this file supplied in every version, at least as long as I juggle them and they are not all merged into one common repository. Perhaps after integrations of all these "initial zeromq version X" PR's we can follow up with a PR to remove this file from older versions. 

On another hand, shouldn't the packages in this gate that are dependent on older zeromq begin complaining during rebuild (if API/ABI did change incompatibly) and so cause revision of their recipes to either explicitly depend on their zeromq-x.y (packages and libs) or bump if upstream updated the code to work with newest zeromq? 

Hipster is the edge expected to bleed ;-\